setup.cfg

```diff
@@ -11,7 +11,7 @@ license = Apache
 license_file = LICENSE
 keywords = physics fitting numpy scipy tensorflow pytorch jax
 project_urls =
-    Documentation = https://scikit-hep.org/pyhf/
+    Documentation = https://pyhf.readthedocs.io/en/v0.5.4/
```

Comment:
Not sure if this is a good idea or not to change _everywhere_, but feels like at least for PyPI and GitHub might be a good idea.